src/main/java/org/myrobotlab/service/InMoov.java

```diff
@@ -1,2483 +0,0 @@
-    }
-  }
-
-  public void moveEyelids(double eyelidleft, double eyelidright) {
-    if (eyelids != null) {
-      eyelids.moveTo(eyelidleft, eyelidright);
-    } else {
-      log.error("moveEyelids - I have a null Eyelids");
-    }
-  }
-
-  public void moveEyes(double eyeX, double eyeY) {
-    if (head != null) {
-      head.moveTo(null, null, eyeX, eyeY, null, null);
-    } else {
-      log.error("moveEyes - I have a null head");
-    }
-  }
-
-  public void moveHand(String which, double thumb, double index, double majeure, double ringFinger, double pinky) {
-    moveHand(which, thumb, index, majeure, ringFinger, pinky, null);
-  }
-
-  public void moveHand(String which, Double thumb, Double index, Double majeure, Double ringFinger, Double pinky, Double wrist) {
-    if (!hands.containsKey(which)) {
-      error("moveHand %s does not exist", which);
-    } else {
-      hands.get(which).moveTo(thumb, index, majeure, ringFinger, pinky, wrist);
-    }
-  }
-
-  public void moveHead(double neck, double rothead) {
-    moveHead(neck, rothead, null);
-  }
-
-  // ---------- canned gestures end ---------
-
-  public void moveHead(double neck, double rothead, double eyeX, double eyeY, double jaw) {
-    moveHead(neck, rothead, eyeX, eyeY, jaw, null);
-  }
-
-  public void moveHead(Double neck, Double rothead, Double rollNeck) {
-    moveHead(neck, rothead, null, null, null, rollNeck);
-  }
-
-  public void moveHead(Double neck, Double rothead, Double eyeX, Double eyeY, Double jaw, Double rollNeck) {
-    if (head != null) {
-      head.moveTo(neck, rothead, eyeX, eyeY, jaw, rollNeck);
-    } else {
-      log.error("I have a null head");
-    }
-  }
-
-  public void moveHeadBlocking(double neck, double rothead) {
-    moveHeadBlocking(neck, rothead, null);
-  }
-
-  public void moveHeadBlocking(double neck, double rothead, Double rollNeck) {
-    moveHeadBlocking(neck, rothead, null, null, null, rollNeck);
-  }
-
-  public void moveHeadBlocking(double neck, double rothead, Double eyeX, Double eyeY, Double jaw) {
-    moveHeadBlocking(neck, rothead, eyeX, eyeY, jaw, null);
-  }
-
-  public void moveHeadBlocking(Double neck, Double rothead, Double eyeX, Double eyeY, Double jaw, Double rollNeck) {
-    if (head != null) {
-      head.moveToBlocking(neck, rothead, eyeX, eyeY, jaw, rollNeck);
-    } else {
-      log.error("I have a null head");
-    }
-  }
-
-  public void moveTorso(double topStom, double midStom, double lowStom) {
-    if (torso != null) {
-      torso.moveTo(topStom, midStom, lowStom);
-    } else {
-      log.error("moveTorso - I have a null torso");
-    }
-  }
-
-  public void moveTorsoBlocking(double topStom, double midStom, double lowStom) {
-    if (torso != null) {
-      torso.moveToBlocking(topStom, midStom, lowStom);
-    } else {
-      log.error("moveTorsoBlocking - I have a null torso");
-    }
-  }
-
-  public void onClassification(TreeMap<String, List<Classification>> classifications) {

```

Comment:
InMoov was copied to InMoov2 before any of this stuff was added to InMoov1...  I think we're throwing away some desired functionality.